### PR TITLE
Update 0007-release-process.md - Adding testing link

### PR DESCRIPTION
**Pull request by vmoutoussamy**
# Describe your changes

Adding link to https://github.com/aquarist-labs/s3gw/blob/main/docs/testing.md in the testing section of the release process document.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] CHANGELOG.md has been updated should there be relevant changes in this PR.

### CODE REVIEW

File: docs/decisions/0007-release-process.md [hunk 1]
```diff
@@ -352,7 +352,8 @@ release candidate was properly validated, version `vX.Y.1` was released.
 
 ## [Testing](testing)
 
-To be expanded in the future, or maybe link to a proper testing document.
+Our project testing is documented [here](/docs/testing.md).
+The list below need to be expanded in the future.
```
Comment (jecluis) [inline docs/decisions/0007-release-process.md]: s/need/needs/

**Review by jecluis — CHANGES_REQUESTED** [review]

**Comment by jecluis** [comment]
@vmoutoussamy please squash the two commits into one. After that I think this is fine to merge.

**Comment by jecluis** [comment]
@vmoutoussamy commit needs to be signed.

**Comment by vmoutoussamy** [comment]
Closing this as it's not really clean and I didn't use a branch in my fork for the PR

**Pull request closed without merging**